app/services/index.js

```diff
@@ -1,2 +1,3 @@
-export { default as voteService } from './topics';
+import voteService from './topics';
```

Comment:
Good point - it makes sense to have one default export. I think we can keep that eslint rule for now. I was hoping we could do something along the lines of https://github.com/reactjs/redux/blob/master/src/index.js#L29...L35 which looks pretty sick hehe.